test/sequential/test-fs-watch.js

```diff
@@ -78,7 +73,9 @@ setImmediate(function() {
   fs.writeFileSync(filepathTwoAbs, 'pardner');
 });
 
-fs.mkdirSync(testsubdir, 0o700);
+var filenameThree = 'newfile.txt';
```

Comment:
I'd just update the variables you're moving. It won't change the size of the diff.